internal/gps/pkgtree/pkgtree.go

```diff
@@ -1046,36 +1043,27 @@ func uniq(a []string) []string {
 // CreateIgnorePrefixTree takes a set of strings to be ignored and returns a
 // trie consisting of strings prefixed with wildcard ignore suffix (*).
 func CreateIgnorePrefixTree(ig map[string]bool) *radix.Tree {
-	var xt *radix.Tree
-
-	// Create a sorted list of all the ignores to have a proper order in
-	// ignores parsing.
-	sortedIgnores := make([]string, len(ig))
```

Comment:
I think the intention here was `make([]string, 0, len(ig))`, but I think that optimization no longer makes sense after moving the filtering up.

Reply:
The bug here was that making with the length only initialized `sortedIgnores` with `len(ig)` empty strings, which made append grow the slice anyways (up to `2*len(ig)`), slowing down the sort and second loop, but the extra empty entries would end up being filtered out.